### ISSUE TITLE
Disourse demo install commands? links? and addons?

### ISSUE THREAD

**Issue by devekko** [fgrehm/devstep#107]
sorry to slam you with issues;

I am trying to get the Discourse demo working and have a few issues

reading from the video, both error, not sure which is in the video, but there are no flags

$ devstep hack -r '-p 9292:9292 --link redis:redis --link postgres:db'
$ devstep hack -r ' -p 9292:9292 --link redis:redis --link postgres:db'

also, how do I enable add-ons?

**Comment by devekko** [comment]
made progress

`git clone https://github.com/discourse/discourse.git
cd discourse
docker run -d --name postgres postgres:9.3 
docker run -d --name redis:redis
devstep hack -p 9292:9292 --link redis:redis --link postgres:db`

inside container
`rake db:create db:migrate`

but Discourse not displaying on host... digging

**Comment by devekko** [comment]
--2015-12-27 00:22:39--  (try: 5)  http://localhost:3000/
Connecting to localhost (localhost)|127.0.0.1|:3000... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

^Z
[1]  + 4970 suspended  wget http://localhost:3000
niccolox@trustyinx:~/Projects/devstep/drupal-core-7|7.x⚡
⇒  curl http://localhost:3000
curl: (52) Empty reply from server

anyway, AWESOME project

would be nice to have updated Discourse info

cheers

**Comment by fgrehm** [comment]
Thanks for the feedback, I'll look into improving docs before the next release.

And yes, the way to go is to use links and run services / project dependencies outside the container. [Addons will be eventually removed](https://github.com/fgrehm/devstep/issues/26#issuecomment-119039576)

**Comment by devekko** [comment]
I actually got passed addon use ok for the Discourse demo

I couldn't get Puma to work within the app... it failed... I guess you have non standard config in your Discourse code?

**Comment by fgrehm** [comment]
I haven't tried running Discourse in a while (and BTW, IIRC that video is about 2 years old now, I need to find the time to improve docs :cry: )

**Comment by fgrehm** [comment]
Yeah, I have just watched the video again and noticed that it comes from the times where the CLI was just a simple bash script :cry: I'll keep this open as a reminder to record new videos when I have a chance. Thanks for the heads up!

**Comment by devekko** [comment]
I just recommended Devstep to a friend and was going to start working on Devstep again myself
